### ISSUE TITLE
Which part shows polarity of sentence

### ISSUE THREAD

**Issue by shanalikhan** [adeshpande3/LSTM-Sentiment-Analysis#1]
Hi, 
I am trying to understand your code but having a bit confusion, you have added all the positive negative sentences in single `ids` and saved and loaded from there.
At the end, it calculate the accuracy after training the NN, i want to know how to check the text  polarity either its positive or negative upon given the sentence

Thanks,
Shan

**Comment by adeshpande3** [comment]
I created a new notebook where I load in a pretrained network, and then set up a couple of functions where you can type in your own sentences, and then see what the model outputs are. Check out the last commit. Let me know if there's still any confusion.

**Comment by shanalikhan** [comment]
Great, Thanks!

**Comment by shanalikhan** [comment]
OKay, i am getting about 40% accuracy on running your first notebook with sample input
Have about 5000 lines with positive and negative sentiment marked...

Any idea why

**Comment by adeshpande3** [comment]
Are you using the pretrained model or training your own? Do you mean you're using 5000 training examples or that the model classified 5000 correctly? How long are you training for?

**Comment by shanalikhan** [comment]
I'm training my own, and setting the labels for training as my own classified. on training function, `[1,0,0]` for positive and other for negative unlike generating the number and setting it in example 
Im training on the default settings, didnt change anything except data.

**Comment by adeshpande3** [comment]
Not particularly sure why you have [1, 0, 0] for the label. Isn't this a binary classification problem? So wouldn't the labels be [1,0] or [0,1]?

**Comment by shanalikhan** [comment]
As my data has 3 so i changed it to [1,0,0] , [0,1,0] [0,0,1] with the change on other variables.
But I tried binary classification same accuracy about 54% for highest iteration.

**Comment by adeshpande3** [comment]
Are you training on your own set of data, or are you training on the movie reviews? If you're using your own set of data, make sure that it's preprocessed correctly, and that most of the words have word vectors associated with them (some will inevitably be <unk>).

With ML models, there's almost an infinite number of things that could be going wrong. You can tune your optimizer/learning rate, change your network architecture, generate your own word vectors instead of using pre-trained ones, etc.
